src/olympia/migrations/994-add-reviewers-admin-group.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `groups` (name, rules, notes, created, modified)
+    VALUES ('Reviewers: Admins', 'Reviews:Admin', '', NOW(), NOW());
+-- Add a few known admins using their user ids from production, as specified in
+-- https://github.com/mozilla/addons-server/issues/7279
+-- If they don't exist on this server, they just will be ignored.
```

Comment:
Unless they're old user accounts (like, more than 3 years old, or whenever the last dump of the prod DB was one) then the user account ids will be different... so you might be giving the access on dev and stage to some other random users who happen to have the same ID as these users on prod.  Maybe we don't care, just pointing it out.  Did we need to do this in the migration?

Reply:
3 of those are old enough to exist and be the same users on dev and stage, the fourth ID is big enough that it doesn't point to an existing user at all. It's not strictly necessary, it could have been done by someone using the UI, I figured it didn't hurt to do it directly in the migration